notebooks/405-paddleOCR-webcam/pre_post_processing.py

```diff
@@ -0,0 +1,459 @@
+import os, os.path
+import sys
+import json
+import urllib.request
+import cv2
+import numpy as np
+import paddle
+import math
+import time
+
+#from IPython import display
+from PIL import Image, ImageDraw, ImageFont
+import copy
+
+import imghdr
+from shapely.geometry import Polygon
+import pyclipper
+
+def DetResizeForTest(data):
+    img = data['image']
+    src_h, src_w, _ = img.shape
+
+    ####resize image to a size multiple of 32 which is required by the network args:
+    ###img(array): array with shape [h, w, c]
+
+    limit_side_len = 960
+    h, w, c = img.shape
+
+    # limit the max side
+    if max(h, w) > limit_side_len:
+        if h > w:
+            ratio = float(limit_side_len) / h
+        else:
+            ratio = float(limit_side_len) / w
+    else:
+        ratio = 1.
+
+    resize_h = int(h * ratio)
+    resize_w = int(w * ratio)
+
+    resize_h = max(int(round(resize_h / 32) * 32), 32)
+    resize_w = max(int(round(resize_w / 32) * 32), 32)
+
+    try:
+        if int(resize_w) <= 0 or int(resize_h) <= 0:
+            return None, (None, None)
+        img = cv2.resize(img, (int(resize_w), int(resize_h)))
+    except:
+        print(img.shape, resize_w, resize_h)
+        sys.exit(0)
+    ratio_h = resize_h / float(h)
+    ratio_w = resize_w / float(w)
+        
+    data['image'] = img
+    data['shape'] = np.array([src_h, src_w, ratio_h, ratio_w])
+    return data
+
```

Comment:
The file is not formatted according to PEP8 (no 2 blank lines between functions)

Reply:
Reviewed the code and delete the 2 blank line cases.